workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/utilities/AddPartyToDistributionList.kt

```diff
@@ -8,6 +8,7 @@ import net.corda.core.node.ServiceHub
 /**
  * Utility function to persist a new entity pertaining to a distribution record.
  * TODO: Add some error handling.
+ * TODO: Don't duplicate pairs of linearId and party.
```

Comment:
This probably doesn't matter for now - before this ever becomes an issue we will transition to DDGs.